src/native/external/llvm-libunwind/src/DwarfInstructions.hpp

```diff
@@ -231,7 +231,7 @@ int DwarfInstructions<A, R>::stepWithDwarf(A &addressSpace, pint_t pc,
         // CFA is the bottom of the current stack frame.
         for (; p < cfa; p += 16) {
           __asm__ __volatile__(".arch armv8.5-a\n"
-                               ".arch_extension memtag\n"
+                               ".arch_extension mte\n"
```

Comment:
Is this a bug in llvm-unwind library?

Reply:
somehow once I used `lse` the `memtag` stopped being recognized. 

The `mte` and `memtag` mean the same thing (memory tagging extensions). It looks like at some point `memtag` was gcc-only and clang supported both `mte` and `memtag`, so `memtag` was preferred (https://reviews.llvm.org/D95996).
I think right now both might be equally supported.

This is confusing and I am not sure if swapping `memtag` for `mte` is the right thing. 
I just want to see if there is anything else will fail.